app/uk/gov/hmrc/trackingconsentfrontend/assets/src/ui/renderBanner.ts

```diff
@@ -50,10 +51,10 @@ const insertBanner = (userPreference: UserPreferences) => {
     insertAfterSkipLink(banner)
 }
 
-const renderBanner = (userPreference: UserPreferences) => {
+export const renderBanner = (_, userPreference: UserPreferences) => {
```

Comment:
Why is this being exported?  I'm assuming it's to test without the `withContentLoaded` condition needing to be met.  If we're taking that approach we should also be testing the default export as that's what other includes are using.

I would personally rather test it in the same way as the settings form is tested - firing the page load and seeing how the unit responds to it.

Reply:
I would rather go the other way and just test the rendering part rather than with the DOM Content Loaded handling, as this functionality can be tested separately.